### PR TITLE
Restrict calibration tool to when the mode is ON.

### PR DESCRIPTION
**Pull request by amrsoll**

### CODE REVIEW

File: octoprint_mrbeam/__init__.py [hunk 1]
```diff
@@ -539,6 +536,22 @@ def get_assets(self):
 			assets['js'].append('js/lib/gremlins.min.js')
 		return assets
 
+	##~~ Helper attributes for different modes
+	# Enable or disable internal support user.
+	@property
+	def support_mode(self):
+		"""Get the support mode"""
+		ret = set_support_mode(self)
```
Comment (andywerner) [inline octoprint_mrbeam/__init__.py]: pls rename to 'check_support_mode'

File: octoprint_mrbeam/__init__.py [hunk 2]
```diff
@@ -1088,6 +1102,8 @@ def printLabel(self):
 	@octoprint.plugin.BlueprintPlugin.route("/engrave_calibration_markers/<string:intensity>/<string:feedrate>", methods=["GET"])
 	# @firstrun_only_access #@maintenance_stick_only_access
 	def engraveCalibrationMarkers(self, intensity, feedrate):
+		if not self.calibration_tool_mode:
+			return NO_CONTENT
```
Comment (andywerner) [inline octoprint_mrbeam/__init__.py]: pls return FORBIDDEN

**Review by andywerner — CHANGES_REQUESTED** [review]
pls add SUPPORT MODE to calibration tool mode

**Comment by amrsoll** [comment]
I'm adding support mode in the usb key recipee

**Review by andywerner — APPROVED** [review]

**Pull request merged**